packages/api/prisma/migrations/20221024073255_guids/migration.sql

```diff
@@ -0,0 +1,8 @@
+-- AlterTable
+ALTER TABLE "articles" ADD COLUMN     "source" TEXT;
+
+-- AlterTable
+ALTER TABLE "authors" ADD COLUMN     "source" TEXT;
+
+-- AlterTable
+ALTER TABLE "images" ADD COLUMN     "origin" TEXT;
```

Comment:
We definitely need one name for all.

What about `peerId`

Reply:
`origin`, `producer` could work too.

Still we should make it connected with the `peers` table with relation and some on-cascades.

@Itrulia wdyt?

Reply:
peerId would clash with the id of the peer for now, would not use that until we have a guid on it.
I like origin